docs/using-bi-for-auth.md

```diff
@@ -49,9 +49,9 @@ https://www.beyondidentity.com/resources/beyond-identity-admin-console-overview
 
    - Email: <email_address>
 
-   - Username: <email_address>
+   - Username: <user_name>
 
-   - Name: <name>
+   - Name: <full_name>
```

Comment:
Let's make sure these match against what the API expects: https://developer-docs-git-v1-beyondidentity.vercel.app/api/v1#tag/Identities/operation/CreateIdentity

Reply:
I was referring to the GUI at that point. I just linked it as an option. Could word it better